examples/todo/main.go

```diff
@@ -207,7 +33,7 @@ func executeQuery(query string, schema graphql.Schema) *graphql.Result {
 
 func main() {
 	http.HandleFunc("/graphql", func(w http.ResponseWriter, r *http.Request) {
-		result := executeQuery(r.URL.Query().Get("query"), schema)
+		result := executeQuery(r.URL.Query().Get("query"), schema.TodoSchema)
 		json.NewEncoder(w).Encode(result)
 	})
```

Comment:
this is not utilising your sample given:
```
	h := handler.New(&handler.Config{
		Schema:     &schema.TodoSchema,
		Pretty:     true,
		GraphiQL:   false,
		Playground: true,
	})

	http.Handle("/graphql", h)
	log.Println("server running on port :8080")
	http.ListenAndServe(":8080", nil)
```

why not take this static out of the example? and why still give GET example. Bu showcasing POST you close the open Issue and its shown by example it can do it :)

Reply:
@maapteh yes, the given example just shows how to connect `handle.New` with `schema.TodoSchema`, in order to show this kind of examples I've created a new repository [`graphql-go/handler-examples`](https://github.com/graphql-go/handler-examples) which we can use to point lib users to in order to show `graphql-go/handler` related examples.

I think is ok to keep the usage of `GET` in `examples/todo` for simplicity of the example itself.

In order to showcase the usage of `POST`, planning to send  a PR for adding a new example `examples/http-post` which will be based from your [PR#573](https://github.com/graphql-go/graphql/pull/573).

Reply:
^ Related PR for adding `examples/http-post`: https://github.com/graphql-go/graphql/pull/575